src/smartbanner.scss

```diff
@@ -4,7 +4,7 @@ $blue: #1474fc;
 
 .smartbanner {
   position: absolute;
-  top: 0;
+  top: -84px;
```

Comment:
@ain Ok, understood re: default styles. I have reinstated them and kept the Android styles.